src/util/server.c

```diff
@@ -703,6 +711,8 @@ int server_setup(const char *name, int flags,
         }
     }
 
+    sss_chain_id_setup(ctx->event_ctx);
```

Comment:
IIUC, this and `sss_chain_id_set_format()` should be moved to 6a54a91beb8767bd3e7df5ce61c128cd985e4feb

Reply:
Fixed